tests/test_agent_service.py

```
@@ -100,6 +100,52 @@ async def _fake_run_async(*args, **kwargs):
     assert "hourly_forecast" not in prompt
 
 
+@pytest.mark.parametrize(
+    "unit,expected_temp,expected_directive",
```

Ruff (0.15.13)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)